### PR TITLE
fix(tests): remove sdd template tests

### PR DESCRIPTION
**Pull request by divyanshiGupta**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [ ] What does this PR do? 
       Remove SDD tests.

- [ ] What issue/task does this PR references?
      https://openshift.io/openshiftio/Openshift_io/plan/detail/1117
- [ ] Are the tests Included?
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by divyanshiGupta** [comment]
[test]

**Comment by joshuawilson** [comment]
What is the issue / work-item for this? We still have spaces that have the SDD template as their choice. 

It may not be an available choice right now, but it is in production.

**Comment by rgarg1** [comment]
@joshuawilson We're not removing the SDD template, but only the tests for that. This was communicated to PM in advance and got a go-ahead in this regard.

**Comment by Raunak1203** [comment]
[test]

**Comment by joshuawilson** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

File: src/tests/specs/quickPreview.spec.ts [hunk 1]
```diff
@@ -72,29 +70,32 @@ describe('Quick preview tests: ', () => {
...
-    await planner.workItemList.clickWorkItem(workItemTitle2);
+    await planner.workItemList.clickWorkItem(workitemname2.title);
     // The description box should not be in edit mode
     expect(await planner.quickPreview.isSaveButtonDisplayed()).toBeFalsy();
   });
 
   it('should close assignee dropdown when clicked outside', async () => {
-    await planner.workItemList.clickWorkItem('Workitem_Title_2');
+    let workitemname = {'title': 'close assignee dropdown'};
+    await planner.workItemList.clickWorkItem(workitemname.title);
```
Comment (Raunak1203) [inline src/tests/specs/quickPreview.spec.ts]: @divyanshiGupta please create this workitem and then click

File: src/tests/specs/quickPreview.spec.ts [hunk 2]
```diff
@@ -72,29 +70,32 @@ describe('Quick preview tests: ', () => {
...
     await planner.quickPreview.assigneeDropdown.clickWhenReady();
     expect(await planner.quickPreview.assigneeDropdownMenu.getAttribute('className')).toContain('show');
     await planner.quickPreview.titleInput.clickWhenReady();
     expect(await planner.quickPreview.assigneeDropdownMenu.getAttribute('className')).not.toContain('show');
   });
 
   it('Should not close description box on outside click if the description is changed', async () => {
-    await planner.workItemList.clickWorkItem('Workitem_Title_2');
+    let workitemname = {'title': 'close description box on outside click'};
+    await planner.workItemList.clickWorkItem(workitemname.title);
```
Comment (Raunak1203) [inline src/tests/specs/quickPreview.spec.ts]: @divyanshiGupta please create this workitem and then click

**Pull request merged**